### PR TITLE
app-editors/ted: bugday 2021-04-03 related fixes

### PR DESCRIPTION
**Pull request by waebbl**
- bump to EAPI 7
fix configure options:
Fix unrecognized configure option --with-GTK
Adjust dependencies
Closes: https://bugs.gentoo.org/470652
Closes: https://bugs.gentoo.org/733734
- use Motif toolkit by default:
Use Motif instead of Gtk-2, which is alternatively
supported as a GUI.

Package-Manager: Portage-3.0.18, Repoman-3.0.3
Signed-off-by: Bernd Waibel <waebbl-gentoo@posteo.net>

### CODE REVIEW

**Review by juippis — COMMENTED** [review]
Then your current 2.23-r1 doesn't compile for me, 
```
make[1]: Leaving directory '/var/tmp/portage/app-editors/ted-2.23-r1/work/Ted-2.23/docBuf'
checking for strerror... config.status: creating makefile
yes
config.status: creating appUtilConfig.h
checking for uname... no
configure: creating ./config.status
yes
Choosing a GUI...
Choosing MOTIF ...
configure: creating ./config.status
config.status: creating makefile
config.status: creating docRtfConfig.h
config.status: creating makefile
config.status: creating tedConfig.h
 * ERROR: app-editors/ted-2.23-r1::gentoo failed (compile phase):
 *   emake failed
```
Although if I lower my `MAKEOPTS` to 1 it passes...

File: app-editors/ted/ted-2.23.ebuild [hunk 1]
```diff
@@ -1,8 +1,8 @@
-# Copyright 1999-2018 Gentoo Foundation
+# Copyright 1999-2021 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=5
-inherit eutils toolchain-funcs xdg-utils
+EAPI=7
```
[the remark this answers is not in the document]
Comment (waebbl) [reply app-editors/ted/ted-2.23.ebuild:4]: Was thinking about revbumping it with the EAPI bump, and didn't do it... Don't know why. I'll change it.

File: app-editors/ted/ted-2.23-r1.ebuild [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+	media-libs/tiff:=
+	sys-libs/zlib
+	virtual/jpeg
+	virtual/libiconv
+	x11-libs/libXft
+	x11-libs/libXpm
+	x11-libs/motif
+"
+DEPEND="${RDEPEND}"
+BDEPEND="virtual/pkgconfig"
```
[the remark this answers is not in the document]
Comment (waebbl) [reply app-editors/ted/ted-2.23-r1.ebuild:31]: fixed in next push

**Comment by waebbl** [comment]
Unfortunately, I was not able to clear the QA warning about jobserver when calling src_install.

**Comment by waebbl** [comment]
gentoo-ci bot seems broken. The output has no hints about this package, but a log of eclass errors

**Review by juippis — APPROVED** [review]

**Pull request closed without merging**